### PR TITLE
Document Android Vulkan layer settings

### PR DESCRIPTION
**Pull request by hevrard**
Bug: b/140620322

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -18,3 +18,29 @@ Please ensure that your setup meets all of the [requirements](../requirements).
 -   The target application must _not_ report any warning or error when run with
     [Vulkan validation layers](https://developer.android.com/ndk/guides/graphics/validation-layer)
     enabled.
+
+## Android Vulkan layers issues
+
+If a Vulkan capture on Android does not terminate properly, AGI may
```
Comment (jakon2048) [inline troubleshooting.md:24]: AGI -> Android GPU Inspector

File: troubleshooting.md [hunk 2]
```diff
@@ -18,3 +18,29 @@ Please ensure that your setup meets all of the [requirements](../requirements).
...
+
+-   `enable_gpu_debug_layers`
+
+-   `gpu_debug_app`
+
+-   `gpu_debug_layers`
+
+-   `gpu_debug_layer_app`
+
+If your app has some Vulkan layer issues after you used AGI, you may
```
Comment (jakon2048) [inline troubleshooting.md:36]: AGI -> Android GPU Inspector

**Review by jakon2048 — COMMENTED** [review]
We should use Android GPU Inspector instead of AGI.

File: troubleshooting.md (hunk 2 above)
Comment (hevrard) [reply troubleshooting.md]: done

File: troubleshooting.md (hunk 1 above)
Comment (hevrard) [reply troubleshooting.md]: done

**Review by pau-baiget — APPROVED** [review]

**Pull request merged**